### DISCUSSION TITLE
Storage is unable to connect after unpause

### DISCUSSION THREAD

**Discussion by alexandre97costa** [supabase/supabase#21399] [Questions]
I unpaused a project and everything got back up normally except for Storage. I cannot create new buckets or view existing ones. I waited 24h before posting this or contacting support, the "Project Status" also states it's "Unable to connect" to storage. This is not the first time I unpause this project, but it's the first time this happens.

I thought this would be an IPv4 problem, but everything else in the project is still acessible and "fine" in my frontend. GET requests to storage respond with a 500 Internal Server Error. I access the storage by making a second client on my frontend, changing the schema of the requested db to "storage". Is this the issue?

```js
export const supabaseClient = createClient(
    PUBLIC_SUPABASE_URL,
    PUBLIC_SUPABASE_ANON_KEY
)

export const supabaseStorage = createClient(
    PUBLIC_SUPABASE_URL,
    PUBLIC_SUPABASE_ANON_KEY,
    {
        db: {
            schema: 'storage'
        }
    }
)
```

Any help would be appreciated, as I didn't find anyone else having the same problem as me.

**Comment by GaryAustin1** [comment]
Did you file a request with support?  That is your best path.

**Reply by alexandre97costa** [reply]
I did contact support yes. I have the free plan, so in before they reply I though it would be a good idea to see if this was human error, or a known issue.

**Reply by GaryAustin1** [reply]
About all I could suggest would be to restart your instance if you have not.

**Comment by alexandre97costa** [comment]
The Storage was back online today, I made no changes so this was just a matter of time.
